src/lib.rs

```diff
@@ -1,8 +1,230 @@
 // Copyright 2019-2023 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
+//! # Actors
+//!
+//! The Filecoin Virtual Machine (FVM) hosts a small[^1] number of _actors_.
+//! These are objects that maintain and mutate internal state, and communicate
+//! by passing messages.
+//!
+//! An example of an actor is the [`cron`](fil_actors_shared::v11::runtime::builtins::Type::Cron)
+//! actor.
+//! Its [internal state](fil_actor_cron_state::v11::State) is a to-do list of
+//! other actors to invoke every epoch.
+//!
+//! See [the Filecoin docs](https://docs.filecoin.io/basics/the-blockchain/actors)
+//! for more information about actors.
+//!
+//! [^1]: The number of [built-in actors](https://docs.filecoin.io/basics/the-blockchain/actors#built-in-actors)
```

Comment:
```suggestion
//! [^1]: The number of [built-in actors](https://github.com/filecoin-project/builtin-actors)
```
As you already linked to the Filecoin docs in the lines directly above, I think the link to the repo makes more sense here and (as an added bonus) is less likely to suffer from link rot.

Reply:
You're right - the redundant link isn't worth it, I'll just remove the link altogether though